docs/tools/lsp-factoryjs/deployment/universal-profile.md

```diff
@@ -329,7 +329,7 @@ await myLSPFactory.UniversalProfile.uploadMetaData(myLSP3MetaData);
 {
   hash: '0x1234...',
   hashFunction: 'keccak256(utf8)',
-  url: 'https://2eff.lukso.dev/ipfs/QmPzUfdKhY6vfcLNDnitwKanpm5GqjYSmw9todNVmi4bqy',
+  url: 'https://ipfs.lukso.network/ipfs/QmPzUfdKhY6vfcLNDnitwKanpm5GqjYSmw9todNVmi4bqy',
```

Comment:
@CallumGrindle it looks like the source text is not up to date. But let's merge the PR anyway as there is other more important stuff.